content/reference/sql/dolt-sql-functions.md

```diff
@@ -417,3 +420,130 @@ SELECT DOLT_MERGE_BASE('feature', 'main');
+queries. For example, you cannot currently alias a table function or join a table function with another table
+or table function. 
+
+## `DOLT_DIFF()`
+
+The `DOLT_DIFF()` table function calculates the differences in a table's data at any two commits in the database.
+Each row in the result set describes how a row in the underlying table has changed between the two commits,
+including the row's values at to and from commits and the type of change (i.e. `added`, `modified`, or `removed`).
+`DOLT_DIFF()` is an alternative to the 
+[`dolt_commit_diff_$tablename` system table](dolt-system-tables#dolt_commit_diff_usdtablename).
+You should generally prefer the system tables when possible, since they have less restrictions on use. 
+However, some use cases, such as viewing a table data diff containing schema changes, can be easier to view 
+with the `DOLT_DIFF` table function.  
+
+The main difference between the results of the `DOLT_DIFF()` table function and the `dolt_commit_diff_$tablename`
+system table is the schema of the returned results. `dolt_commit_diff_$tablename` generates the resulting schema
+based on the table's schema at the currently checked out branch. `DOLT_DIFF()` will use the schema at the `from_commit`
+for the `from_` columns and the schema at the `to_commit` for the `to_` columns. This can make it easier to view 
+diffs where the schema of the underlying table has changed. 
+
+Note that the `DOLT_DIFF()` table function currently has restrictions on how it can be used in queries. It does not
+support aliasing or joining with other tables, and argument values must currently be literal values. 
+
+### Options
+
+```sql
+DOLT_DIFF(<tablename>, <from_revision>, <to_revision>)
+```
+The `DOLT_DIFF()` table function takes three required arguments:
+* `tablename`  —  the name of the table containing the data to diff
+* `from_revision`  — the revision of the table data for the start of the diff. This may be a commit, tag, branch name, or other revision specifier (e.g. "main~").
+* `to_revision`    — the revision of the table data for the end of the diff. This may be a commit, tag, branch name, or other revision specifier (e.g. "main~").
+
+### Schema 
+
+```text
++------------------+----------+
+| field            | type     |
++------------------+----------+
+| from_commit      | TEXT     |
+| from_commit_date | DATETIME |
+| to_commit        | TEXT     |
+| to_commit_date   | DATETIME |
+| diff_type        | TEXT     |
+| other cols       |          |
++------------------+----------+
+```
+
+The remaining columns are dependent on the schema of the user table as it existed at the `from_commit` and at 
+the `to_commit`. For every column `X` in your table at the `from_commit` revision, there is a column in the result 
+set named `from_X`. Likewise, for every column 'Y' in your table at the `to_commit` revision, there is a column
+in the result set named `to_Y`. This is the major difference between the `DOLT_DIFF()` function and the
```

Comment:
"this is the major diff" doesn't really tell me what the major diff is. Be more explicit here